src/commands/GameIntegration/clashofclans.ts

```diff
@@ -29,9 +28,9 @@ const kFilterSpecialCharacters = /[^A-Z0-9]+/gi;
 	subCommands: ['player', { input: 'clan', default: true }]
 })
 export class UserPaginatedMessageCommand extends PaginatedMessageCommand {
-	public async clan(message: GuildMessage, args: PaginatedMessageCommand.Args) {
+	public async clan(message: Message, args: PaginatedMessageCommand.Args) {
 		const { t } = args;
-		const clan = await args.pick('string');
+		const clan = (await args.rest('string')).slice(6);
```

Comment:
Why are you getting everything and only using whats after the 6th char?

Reply:
slicing off `clan ` subcommand

Reply:
Wait, what?

Reply:
Wait what seconded

Reply:
Wait what thirded

Reply:
I can't repro it anymore. Reverted this.